frameworks/helloworld/tests/test_mount_volumes.py

```diff
@@ -1,19 +1,67 @@
+import json
 import logging
+import os
+import time
 
-import os.path
 import pytest
 
 import sdk_cmd
 import sdk_hosts
 import sdk_install
 import sdk_plan
 import sdk_tasks
-import sdk_utils
 from tests import config
 
+import dcos_launch
+import dcos_test_utils
+
 log = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope='session')
+def mount_volumes():
```

Comment:
Why is this added as a fixture in the test code?  Shouldn't it be a script called after cluster provisioning succeeds and run once to provide MOUNT volumes for the cluster?  Then tests can expect a cluster with MOUNT volumes.

Reply:
Moved the code to `tools` and added a hook to run it in CI instead